### ISSUE TITLE
Error: no such table: X

### ISSUE THREAD

**Issue by Skorlok** [EvoEsports/minicontrol#2]
Hi,
I'm trying to make a docker container for Minicontrol, but I have some bugs with the database, it seems to Players and Records table are not found

```
minicontrol  | drizzle-kit: v0.20.10
minicontrol  | drizzle-orm: v0.29.3
minicontrol  | 
minicontrol  | Reading config file '/app/controller/drizzle.config.ts'
minicontrol  | 0 tables
minicontrol  | 
minicontrol  | 
minicontrol  | No schema changes, nothing to migrate 😴
minicontrol  | Starting MiniControl...
minicontrol  | Connecting to Trackmania Dedicated server at dedicated:4999
minicontrol  | Connected to Trackmania Dedicated server.
minicontrol  | Plugin database loaded.
minicontrol  | Running Migrates...
minicontrol  | Database connected.
minicontrol  | Plugin chat loaded.
minicontrol  | Plugin admin loaded.
minicontrol  | Plugin players loaded.
minicontrol  | Plugin maps loaded.
minicontrol  | Plugin bestcps loaded.
minicontrol  | Plugin mapwidget loaded.
minicontrol  | Plugin tmx loaded.
minicontrol  | Plugin announces loaded.
minicontrol  | Plugin tmnf_talimit not loaded. Game is not TmForever.
minicontrol  | Plugin tmnf_coppers not loaded. Game is not TmForever.
minicontrol  | Plugin tmnf_freezone not loaded. Game is not TmForever.
minicontrol  | Plugin tmnf_ui not loaded. Game is not TmForever.
minicontrol  | Plugin dedimania not loaded. Game is not TmForever.
minicontrol  | Plugin debugtool loaded.
minicontrol  | Plugin votes loaded.
minicontrol  | Plugin funcommands loaded.
minicontrol  | Plugin records loaded.
minicontrol  | Plugin togglechat not loaded. Game is not TmForever.
minicontrol  | [525.44ms] Startup
minicontrol  | MiniControl started successfully.
minicontrol  | Welcome to MINIcontrol v2024-01-11!
minicontrol  | 15 |   logger;
minicontrol  | 16 |   exec(query) {
minicontrol  | 17 |     this.client.exec(query);
minicontrol  | 18 |   }
minicontrol  | 19 |   prepareQuery(query, fields, executeMethod, customResultMapper) {
minicontrol  | 20 |     const stmt = this.client.prepare(query.sql);
minicontrol  |                       ^
minicontrol  | SQLiteError: no such table: player
minicontrol  |  errno: 1
minicontrol  | 
minicontrol  |       at prepare (bun:sqlite:193:19)
minicontrol  |       at prepareQuery (/app/controller/node_modules/drizzle-orm/bun-sqlite/session.js:20:18)
minicontrol  |       at prepare (/app/controller/node_modules/drizzle-orm/sqlite-core/query-builders/select.js:571:19)
minicontrol  |       at /app/controller/node_modules/drizzle-orm/sqlite-core/query-builders/select.js:583:12
minicontrol  |       at execute (/app/controller/node_modules/drizzle-orm/sqlite-core/query-builders/select.js:591:19)
minicontrol  |       at then (/app/controller/node_modules/drizzle-orm/query-promise.js:21:12)
minicontrol  |       at processTicksAndRejections (:61:77)
```

**Comment by Chris92de** [comment]
We encountered the same issue and are in the process of publishing both a minicontrol image as well as a proper TMNF docker image on the docker hub.
Please be patient :)

**Comment by reaby** [comment]
fixed with #3 
also image available at:
https://hub.docker.com/r/evoesports/minicontrol/tags

**Comment by ItsPhenom** [comment]
And here the TM Forever server image:
https://hub.docker.com/r/evoesports/trackmaniaforever
